tensorbay/opendataset/COCO2017/loader.py

```diff
@@ -0,0 +1,227 @@
+                data = Data(image_path)
+                label = data.label
+                label.box2d = []
+                label.multi_polygon = []
+                label.keypoints2d = []
+                label.rle = []
+
+                image_name = os.path.splitext(os.path.basename(image_path))[0]
+                image_id = int(image_name)
+
+                categories = task_information["categories"]
+                _get_instance_label(
+                    task_information["instances_annotations"], image_id, categories, label
+                )
+                _get_person_keypoints_label(
+                    task_information["person_keypoints_annotations"], image_id, categories, label
+                )
+                _get_panoptic_box2d_label(
+                    task_information["panoptic_annotations"], image_id, categories, label
+                )
+                _get_panoptic_mask_label(
+                    annotation_path,
+                    segment_name,
+                    image_name,
+                    task_information["panoptic_annotations"],
+                    label,
+                )
+                segment.append(data)
+
+    return dataset
+
+
+def _get_annotations(annotation_path: str, segment_name: str) -> Dict[str, Any]:
+    task_information: Dict[str, Any] = {}
+    for task in ["instances", "person_keypoints", "panoptic"]:
+        with open(os.path.join(annotation_path, f"{task}_{segment_name}2017.json"), "r") as js:
+            file_json = json.load(js)
+
+            task_annotation: Dict[int, Any] = {}
+            annotations = file_json["annotations"]
+            for annotation in annotations:
+                image_id = annotation["image_id"]
+                if image_id in task_annotation.keys():
+                    task_annotation[image_id].append(annotation)
+                else:
+                    task_annotation[image_id] = [annotation]
+            task_information[f"{task}_annotations"] = task_annotation
+
+            if task == "panoptic":
+                categories: Dict[int, str] = {}
+                for category in file_json["categories"]:
+                    categories[category["id"]] = f"{category['supercategory']}.{category['name']}"
+                task_information["categories"] = categories
+
+    return task_information
+
+
+def _get_instance_label(
+    ins_annotations: Dict[int, Any], image_id: int, categories: Dict[int, str], label: Label
+) -> None:
+    if image_id not in ins_annotations.keys():
+        return None
+    for annotation in ins_annotations[image_id]:
+        category = categories[annotation["category_id"]]
+        label.box2d.append(LabeledBox2D.from_xywh(*annotation["bbox"], category=category))
+        if annotation["iscrowd"] == 0:
+            points = []
+            for coordinates in annotation["segmentation"]:
+                points.append(zip(coordinates[::2], coordinates[1::2]))
```

Comment:
Use list comprehension? 
`[zip(coordinates[::2], coordinates[1::2]) for coordinates in annotation["segmentation"]]`